sqlmesh/core/renderer.py

```diff
@@ -291,7 +302,7 @@ def render(
             The rendered expression.
         """
         cache_key = _dates(start, end, latest)
-        skip_cache = bool(snapshots or expand)
+        skip_cache = bool(snapshots or table_mapping or expand)
```

Comment:
@tobymao I don't get why we skip cache here on snapshots, etc. We don't cache after table resolution anyway.

Reply:
if these are not needed, can you remove them @crericha

Reply:
Confirmed not needed anymore. Removed in the latest commit to this PR.